.github/workflows/test.yml

```diff
@@ -61,114 +54,146 @@ jobs:
           name: replay-driver
           path: macOS-recordreplay.so
   test0:
-    name: Test stripe 0
+    name: End-to-end tests (0)
     runs-on: macos-latest
-    needs: [build, download-browser, download-node, download-driver]
-    timeout-minutes: 15
+    needs: [build-server, download-browser, download-node, download-driver]
+    timeout-minutes: 20
     steps:
       - uses: actions/checkout@v2
       - uses: actions/download-artifact@v2
+      - run: mv node_modules/node_modules.tgz ./
+      - run: tar -xzf node_modules.tgz
+      - uses: ./.github/actions/setup-next
```

Comment:
I don't propose this change for here but i'm curious if we should bundle all of these into our test custom action since it's already a composite action now and it's not really reusable outside this repo anyway.

Reply:
Yeah, I actually did that, and then I realized that you can't call another locally defined action from a locally defined action. More specifically you can't use uses inside of an action (although it seems like you can as long as it's not a local uses? Idk.) More details here: https://github.com/actions/runner/issues/646. But I do think the untarring can be consolidated, so I'll move that.